### ISSUE TITLE
ValidationError "is not a valid load balancer ARN" even when ARNs are OK

### ISSUE THREAD

**Issue by flaviostutz** [docker/compose-cli#874]
**Description**

When using an external LoadBalancer, during initial checks an error is raised telling the ARN is not valid, but the ARN was double checked and is OK.

**Steps to reproduce the issue:**
1. Create a docker-compose.yml file with reference `x-aws-loadbalancer: arn:aws:elasticloadbalancing:us-west-2:bbbbbbbbbbbbb:loadbalancer/app/poc2-stutz-com-br-default/aaaaaaaaaaaaaaa`
2. Create a context that points to region us-west-2 with `docker context create ecs`
3. Create a AWS SDK profile with the credentials needed for compose to work
4. Execute `docker compose up`
5. Check logs for error

**Describe the results you received:**
```
DEBU[0003] Check if LoadBalancer exists: arn:aws:elasticloadbalancing:us-west-2:bbbbbbbbbbbbb:loadbalancer/app/poc2-stutz-com-br-default/aaaaaaaaaaaaaaa
ValidationError: 'arn:aws:elasticloadbalancing:us-west-2:bbbbbbbbbbbbb:loadbalancer/app/poc2-stutz-com-br-default/aaaaaaaaaaaaaaa' is not a valid load balancer ARN
        status code: 400, request id: 3505dbe5-1369-4563-9ba2-cd3d873e78fb
```

**Describe the results you expected:**
I expected the existing LoadBalancer to be found because ARN is valid and LoadBalancer does exist as we can see on AWS console in the specific region.

**Additional information you deem important (e.g. issue happens only occasionally):**
In some cases this was working on my machine, but after changing AWS regions it stopped working.

**Output of `docker context show`:**  
You can also run `docker context inspect context-name` to give us more details but don't forget to remove sensitive content.

```
(paste your output here) 
```

**Output of `docker info`:**

```
Client:
 Debug Mode: false
 Plugins:
  app: Docker Application (Docker Inc., v0.8.0)
  buildx: Build with BuildKit (Docker Inc., v0.4.2-tp-docker)
  scan: Docker Scan (Docker Inc., v0.3.4)

Server:
 Containers: 0
  Running: 0
  Paused: 0
  Stopped: 0
 Images: 401
 Server Version: 19.03.13
 Storage Driver: overlay2
  Backing Filesystem: extfs
  Supports d_type: true
  Native Overlay Diff: true
 Logging Driver: json-file
 Cgroup Driver: cgroupfs
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
 Swarm: active
  NodeID: vfw3bft5qkwxp8f2x8me35t5d
  Is Manager: true
  ClusterID: 5okhxug6ylqnec72ia0fa3ef7
  Managers: 1
  Nodes: 1
  Default Address Pool: 10.0.0.0/8
  SubnetSize: 24
  Data Path Port: 4789
  Orchestration:
   Task History Retention Limit: 5
  Raft:
   Snapshot Interval: 10000
   Number of Old Snapshots to Retain: 0
   Heartbeat Tick: 1
   Election Tick: 10
  Dispatcher:
   Heartbeat Period: 5 seconds
  CA Configuration:
   Expiry Duration: 3 months
   Force Rotate: 0
  Autolock Managers: false
  Root Rotation In Progress: false
  Node Address: 192.168.65.3
  Manager Addresses:
   192.168.65.3:2377
 Runtimes: runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: 8fba4e9a7d01810a393d5d25a3621dc101981175
 runc version: dc9208a3303feef5b3839f4323d9beb36df0a9dd
 init version: fec3683
 Security Options:
  seccomp
   Profile: default
 Kernel Version: 4.19.76-linuxkit
 Operating System: Docker Desktop
 OSType: linux
 Architecture: x86_64
 CPUs: 5
 Total Memory: 7.78GiB
 Name: docker-desktop
 ID: M2XC:7THC:EZRF:QYS5:AXAI:HM3J:QTMA:RF3G:EUNT:OOV7:X2ED:EPYM
 Docker Root Dir: /var/lib/docker
 Debug Mode: false
 HTTP Proxy: gateway.docker.internal:3128
 HTTPS Proxy: gateway.docker.internal:3129
 Registry: https://index.docker.io/v1/
 Labels:
 Experimental: true
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false
 Product License: Community Engine
```

**Additional environment details (AWS ECS, Azure ACI, local, etc.):**
ECS

**Comment by flaviostutz** [comment]
I found the cause of this behavior, but it still looks like a bug.

In my aws cli profile, the **default** region (~/.aws/config) was set to another region. When I set the "aws cli default region" to the same region as "docker compose context", it works as expected, but I think we should circumvent this behavior (as the aws region was explicitly set in command line) or at least show an specific message about this.
